daemon/src/engine/renderer/tr_image_exr.cpp

```diff
@@ -28,7 +28,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 #include "openexr/half.h"
 
 	void LoadRGBEToFloats( const char *name, float **pic, int *width, int *height );
-
+	// Unused, forward declared in tr_bsp.cpp
```

Comment:
I've seen this one many times and decided to keep it because it might be useful. I'm not attached to it though.